app/js/templates/userTable.hbs

```diff
@@ -2,6 +2,7 @@
     <div class="user_item">
         <p class="name">Name: {{name}}</p>
         <p class="email">Email: {{email}}</p>
+        <button class="btn btn-success" id="edit-btn">Edit User</button>
```

Comment:
btn-succes is green.... Wrong UX. please change

Reply:
you will have multiple reoccuring id's. Id's should only have one place in the file.

Reply:
the button has padding automatically set and this is making the button uneccesarily large. Taking up too much space.

Reply:
UX....? this isn't final and therefore not MVP

We have spoken about the ID's. It is necessary to hook into the click event.

I like my buttons with padding...

Reply:
removed ID